packages/gatsby-plugin-page-creator/README.md

```diff
@@ -15,6 +15,10 @@ With this plugin, _any_ file that lives in the `src/pages` folder (or subfolders
 - `_*`
 - `.*`
 
+NOTE: also excludes `test.js` and `spec.js` (or .ts, .jsx, .tsx)
```

Comment:
This should be fixed in https://github.com/gatsbyjs/gatsby/pull/11862 so I think it's safe to drop this line

Reply:
Done!